src/types/TempTodo.ts

```diff
@@ -0,0 +1,5 @@
+export interface TempTodo {
+  userId: number;
+  title: string;
+  completed: boolean;
+}
```

Comment:
FYI you can use utility type here to write more advanced TS code)
check this out)

https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
thx, I didn't know that before)